Assets/Plugins/Source/Editor/Internal/PackageFileUtils.cs

```diff
@@ -199,23 +199,6 @@ public static List<FileInfoMatchingResult> GetFileInfoMatchingPackageDescription
                             Debug.LogWarning("Copy error for file (" + srcToDestKeyValues.src + ") :" + srcToDestKeyValues.sha1_mismatch_error);
                         }
                     }
-                    if (!EmptyPredicates.IsEmptyOrNull(srcToDestKeyValues.signWithDefaultCertificate))
-                    {
-                        
-                        if (toolsSection != null)
-                        {
-                            if (toolsSection.GetCurrentConfig() == null)
-                            {
-                                toolsSection.LoadConfigFromDisk();
-                            }
-                            var pathToSignTool = toolsSection.GetCurrentConfig().pathToEACIntegrityTool;
-                            var pathToDefaultCertificate = toolsSection.GetCurrentConfig().pathToDefaultCertificate;
-                        }
-                        else
-                        {
-                            Debug.LogError("No tools configuration set!");
-                        }
-                    }
```

Comment:
This code section was removed because it does nothing. If you evaluate it carefully, you'll see that it creates a variable `pathToSignTool` and `pathToDefaultCertificate`, but doesn't do anything with either of them. Lack of documentation around this makes it unclear what the intended functionality is, so it was removed to avoid future confusion - but remains in the `git` record for historical context.

Reply:
Nope. Eventually we'll need to circle back to this to resolve. It's not mentioned in the docs though, so until we fully implement it and reference it in the documentation, it's safe for it to quietly die (particularly because we have the history tracked via git).